ios/Podfile.lock

```diff
@@ -0,0 +1,33 @@
+PODS:
```

Comment:
```
Resolving dependencies of `Podfile`
    [!] CocoaPods could not find compatible versions for pod "GoogleMaps":
      In snapshot (Podfile.lock):
        GoogleMaps (= 3.5.0)
      In Podfile:
        google_maps_flutter (from `.symlinks/plugins/google_maps_flutter/ios`) was resolved to 0.0.1, which depends on
          GoogleMaps
    It seems like you've changed the constraints of dependency `GoogleMaps` inside your development pod `google_maps_flutter`.
    You should run `pod update GoogleMaps` to apply changes you've made.
```
got this error locally trying to run. did you have to do some setup work to make it run?